lib/template/mkcol.js

```diff
@@ -0,0 +1,18 @@
+import prop from './prop';
+
+export default function mkcol(object) {
+  return `<d:mkcol xmlns:c="urn:ietf:params:xml:ns:caldav"
+              xmlns:card="urn:ietf:params:xml:ns:carddav"
+              xmlns:cs="http://calendarserver.org/ns/"
+              xmlns:d="DAV:">
+    <d:set>
+      <d:prop>
+      	<d:resourcetype>
+      	  <d:collection/>
+          <card:addressbook/>
```

Comment:
`mkcol` isn't addressbook specific. Can we have a generic template for `mkcol` and maybe a more specific one for making a vcard collection that uses the generic one?